Documentation/ColumnsConfig/Type/Color/Index.rst

```diff
@@ -19,6 +19,12 @@ Color
 
 The TCA type :php:`color` should be used to render a JavaScript-based color picker.
 
+..  versionadded:: 13.0
+
+:ref:`Color palettes <t3tsconfig:pagecolorpalettes>` can be defined via
+:ref:`page TSconfig <t3tsconfig:setting-page-tsconfig>`. This way, for example,
+colors defined in a corporate design can be made accessible in an easy way.
```

Comment:
```suggestion
:ref:`page TSconfig <t3tsconfig:setting-page-tsconfig>`. With this method, the user can pick predefined corporate design colors with a click instead of typing in RGB values.
```
Maybe like this?

Reply:
Yes. I would say "select" rather than "pick" (maybe then you could say "a colorpicker" rather than "a click") but otherwise it looks fine to me.

Reply:
Done